src/Resources/skeleton/crud/templates/edit.tpl.php

```diff
@@ -0,0 +1,24 @@
+<?= $helper->getHeadPrintCode('Edit '.$entity_class_name) ?>
+
+{% block stylesheets %}
+<style>
+    .example-wrapper { margin: 1em auto; max-width: 1018px; width: 95%; }
+    .example-wrapper form div { margin: 1em 0; }
+    .example-wrapper form label { display: block; }
+    .example-actions form, .example-actions a { display: inline-block; margin: 0 2px; }
+</style>
+{% endblock %}
+
+{% block body %}
+    <div class="example-wrapper">
```

Comment:
I don't think we should add this class, or the CSS above it. I *do* like the idea of the CRUD not looking *horrible* initially... but the user WILL need to customize this and make it their own. So, we should not add any extra stuff that they will need to remove (or, we should be VERY minimalistic).

Actually, I talked to Javier, and we both agree that we should add a few *basic* Bootstrap classes  e.g .`<table class="table"` and `<button class="btn">`. But, I think that's all we should include - we shouldn't package our own CSS.

Reply:
ok good. It was just my experiment :) will add simple bootstrap classes